### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by stineb** [stineb/rsofun#7]
Could not build package due to:
`ERROR: a 'NAMESPACE' file is required`

**Comment by jmaspons** [comment]
NAMESPACE is generated from the roxygen comments. By default, the roxygen is not updated on «Install and Restart». Run Build > More > Document (Ctrl + Shift + D)

**Comment by stineb** [comment]
This error is now resolved. But build failure on appveyor now happens as described there:
```
Error: (converted from warning) package 'Biobase' is not available (for R version 3.5.1 Patched)
Execution halted
Command exited with code 1
7z a failure.zip *.Rcheck\*
7-Zip 18.05 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2018-04-30
```

**Comment by khufkens** [comment]
Fixed as of newer versions
